app/controllers/api/professor/accommodations_controller.rb

```diff
@@ -0,0 +1,77 @@
+# frozen_string_literal: true
+
+module Api
+  module Professor
+    # Accommodation controls for exams.
+    class AccommodationsController < ProfessorController
+      before_action :find_accommodation, only: [:update, :destroy]
+      before_action :find_exam_and_course
+      before_action :require_prof_reg
+
+      def index
+        render json: {
+          accommodations: @exam.accommodations.map do |acc|
+            reg = acc.registration
+            user = reg.user
+            {
+              id: acc.id,
+              startTime: acc.new_start_time,
+              extraTime: acc.percent_time_expansion,
```

Comment:
We should probably systematically rename `extraTime` to at least `percentExtraTime` everywhere